scripts/variantstore/wdl/extract/create_cohort_extract_data_table.py

```diff
@@ -57,7 +59,9 @@ def execute_with_retry(label, sql):
   start = time.time()
   while len(retry_delay) > 0:
     try:
-      query = client.query(sql)
+      labelValue = label.replace(" ","-").strip().lower()
```

Comment:
rename the parameter to this function (above) to be "query_name" or something since label means something else now...